src/main/java/org/opensearch/ad/task/ADTaskManager.java

```diff
@@ -281,21 +294,60 @@ protected void forwardDetectRequestToCoordinatingNode(
         DiscoveryNode node,
         ActionListener<AnomalyDetectorJobResponse> listener
     ) {
-        TransportRequestOptions option = TransportRequestOptions
-            .builder()
-            .withType(TransportRequestOptions.Type.REG)
-            .withTimeout(requestTimeout)
-            .build();
         transportService
             .sendRequest(
                 node,
                 ForwardADTaskAction.NAME,
                 new ForwardADTaskRequest(detector, detectionDateRange, user, adTaskAction),
-                option,
+                getTransportRequestOptions(),
                 new ActionListenerResponseHandler<>(listener, AnomalyDetectorJobResponse::new)
             );
     }
 
+    /**
+     * Forward AD task to coordinating node
+     *
+     * @param adTask AD task
+     * @param adTaskAction AD task action
+     * @param transportService transport service
+     * @param listener action listener
+     */
+    protected void forwardADTaskToCoordinatingNode(
+        ADTask adTask,
+        ADTaskAction adTaskAction,
+        TransportService transportService,
+        ActionListener<AnomalyDetectorJobResponse> listener
+    ) {
+        transportService
+            .sendRequest(
+                getCoordinatingNode(adTask),
+                ForwardADTaskAction.NAME,
+                new ForwardADTaskRequest(adTask, adTaskAction),
+                getTransportRequestOptions(),
```

Comment:
Copy from https://github.com/opensearch-project/anomaly-detection/pull/58#discussion_r640948167

> How about create this as a field so that it can be reused?

The request timeout parameter may be updated for new request, so will create transport request for every request.

Reply:
Can you create a clusterservice consumer?

Reply:
The request timeout has cluster service consumer now. You mean create a consumer for transport request as well?